src/sprout/Helpers/HttpReq.php

```diff
@@ -195,9 +195,11 @@ function($curl, $header) use (&$headers)
             curl_setopt($ch, CURLOPT_HTTPAUTH, CURLAUTH_BASIC);
         }
 
+        $cert_path = $opts['cacert_path'] ?? (APPPATH . 'cacert.pem');
```

Comment:
Did we also want this custom CA dealio for fopen implementation?

Reply:
I'll add that next time :D